baseframe/static/js/baseframe-material.js

```diff
@@ -249,6 +249,43 @@ window.Baseframe.Forms = {
         }
       }
     })
+  },
+  /* Takes 'formId' and 'errors'
+     'formId' is the id attribute of the form for which errors needs to be displayed
+     'errors' is the wtf form validation errors expected in the following format
+      {
+        "title": [
+          "This field is required."
+        ]
+        "email": [
+          "Not a valid email."
+        ]
+      }
+    For each error, a 'p' tag is created if not present and
+    assigned the error value as its text content.
+    The field wrapper and field are queried in the DOM
+    using the unique form id. And the newly created 'p' tag
+    is inserted in the DOM below the field.
+  */
+  showValidationErrors: function(formId, errors) {
+    Object.keys(errors).forEach(function(fieldName) {
+      if (Array.isArray(errors[fieldName])) {
+        var form = document.getElementById(formId);
```

Comment:
This need not run inside a loop, right? That way it can be fetched once before `Object.keys` is iterated on? Since it's going to be the same for all the errors.

Reply:
Fixed